modules/machineconfig-modify-registry.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * installing/post_installation_configuration/machine-configuration-tasks.adoc
+
+[id="machineconfig-modify-registry_{context}"]
+= Configuring container image registry settings
+
+Settings that define the registries {product-title} uses to get container images
+are held in the `/etc/containers/registries.conf` by default. In that file,
```

Comment:
"in the `/etc/containers/registries.conf` file"

Reply:
Done